src/super_gradients/training/models/segmentation_models/segformer.py

```diff
@@ -326,12 +328,17 @@ def __init__(
         :param overlap_patch_stride:  the patch stride of the overlapping patch embedding in each stage
         :param overlap_patch_pad:  the patch padding of the overlapping patch embedding in each stage
         :param in_channels:  number of input channels
+        :param sliding_window_crop_size:  (width, height) the crop size to take from the image for forward with sliding window
+        :param sliding_window_stride:  (rows, cols) the stride size between crops for forward with sliding window
```

Comment:
I strongly suggest not to mix parameters taking (rows/cols) and (cols/rows) in a single function.
No double, it's helpful to have a docstring, but why we use (rows/cols) in one parameter and (cols/rows) in a other one?

Reply:
Fixed on 2821ee.